### ISSUE TITLE
Please provide maintainable list of NDK codename-revision pairs

### ISSUE THREAD

**Issue by basilgello** [android/ndk#2042]
In certain situations like F-Droid builds it is necessary to have a mapping of NDK codename (r26d) to revision (26.3.xxxxx), since sdkmanager from Android SDK does not know MDK codenames.

I made a quick one-liner to retrieve NDK revision from released tags in this repo:

```
wget \
  -qO- \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  'https://api.github.com/repos/android/ndk/releases' |
  jq \
   -r \
   '.[] | select(.tag_name == "r27-beta1") | .body | match("ndkVersion \"(.*)\"").captures[0].string'
```

but tagged releases start from `r26`.

To provide single source of truth on this matter, it would be best to have releases tagged for older NDKs up to r11 but I am not sure if it possible to back-date Github releases so they dont become the 'latest' one. Another option is to have automatically generated JSON/YAML/TOML file with mappings like:

```
r26=26.0.11112
```

@DanAlbert What do you think?

**Comment by DanAlbert** [comment]
You mean this? https://github.com/android/ndk/wiki (and https://github.com/android/ndk/wiki/Unsupported-Downloads for the unsupported releases)

I keep asking Studio to handle our version scheme. I think they agree, there's just no capacity for making it happen.

**Comment by basilgello** [comment]
Yes, this one. With more sophisticated regexp-ing it may be done but I guess some automatic step generating the file is better

**Comment by DanAlbert** [comment]
I guess I'm not really sure what you're asking me to do. The data is already published.

**Comment by DanAlbert** [comment]
Closing for lack of feedback. If you can clarify what you're asking for we can reopen, but afaict this is already fixed.

**Comment by basilgello** [comment]
Sorry for delays! @linsui posted a solution provided by F-Droid: https://gitlab.com/fdroid/android-sdk-transparency-log
Namely, https://gitlab.com/fdroid/android-sdk-transparency-log/-/raw/master/checksums.json
